lib/custom_widgets/genres.dart

```diff
@@ -15,10 +15,10 @@ class WidgetedGenres extends StatelessWidget {
   List<Widget> _createWidgets() {
```

Comment:
also rename this to something more specific

Reply:
Done